xml/abstract_commands.xml

```diff
@@ -26,14 +26,25 @@
         and must support read and write access to all GPRs when the selected hart is halted.
         Debug Modules may optionally support accessing other registers,
         or accessing registers when the hart is running.
-        Each individual register (aside from GPRs) may be supported differently
-        across read, write, and halt status.
+        It is recommended that if one register in a group is accessible, then
+        all registers in that group are accessible, but each individual
+        register (aside from GPRs) may be supported differently across read,
+        write, and halt status.
+
+        Registers may not be accessible if they wouldn't be accessible by M
```

Comment:
```suggestion
        Registers might not be accessible if they wouldn't be accessible by M
```
"may not" is ambiguous (sometimes meaning "shall not" and sometimes meaning "might not").